openslides/users/views.py

```diff
@@ -162,6 +164,55 @@ def destroy(self, request, *args, **kwargs):
         self.perform_destroy(instance)
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    def update(self, request, *args, **kwargs):
```

Comment:
As discussed with @normanjaeckel 
First, please fix the 'add' case only (without complex diff code).
Let us move the 'remove' case to 2.2.